### ISSUE TITLE
Disable rsync daemon

### ISSUE THREAD

**Issue by karras** [adfinis-sygroup/ansible-role-console#16]
##### ISSUE TYPE

 - Bug Report

##### ANSIBLE VERSION

```
n/a
```

##### SUMMARY

The `rsync` package on Ubuntu/Debian (and maybe also on EL) enables the rsync daemon by default. Why the heck would you need the a daemon on every system?! This role should therefore disable the rsync daemon after the package installation to prevent unnecessary service startups, even if the daemon fails by default due to the missing start condition (config file).

##### STEPS TO REPRODUCE

```yaml

```

##### ACTUAL RESULTS

```

```

**Comment by lordfolken** [comment]
In order for it to actually start /etc/rsyncd.conf must exist. Its a precondition in the rsync's service file:
```init
[Unit]
Description=fast remote file copy program daemon
ConditionPathExists=/etc/rsyncd.conf
```

```bash
● rsync.service - fast remote file copy program daemon
   Loaded: loaded (/lib/systemd/system/rsync.service; enabled; vendor preset: enabled)
   Active: inactive (dead)
Condition: start condition failed at Fri 2019-12-20 14:10:12 CET; 5 days ago
           └─ ConditionPathExists=/etc/rsyncd.conf was not met
``` 
So its not actually launched. (The config file isn't supplied by the package, at least on stretch/buster).

**Comment by karras** [comment]
Yep, as I've described above in the last sentence. ;) Though IMHO this is not a nice approach. Either have a service enabled and use it or simply disable it.

**Comment by lordfolken** [comment]
Disabling it in this role, would actually potentially upset another role where it is being used/enabled. We should not change defaults.

**Comment by karras** [comment]
I'd rather question if we can't remove it at all in this role. rsync is only used on a small amount of systems.

**Comment by lordfolken** [comment]
I would also favor that. While its a pretty basic utillity, it should only be installed if actually required.

**Comment by lordfolken** [comment]
Looking through the role, i would question the entire thing. 
 * this thing tries to install a "workstation" like environment
 * it isn't doing a good job for that
 * way to many packages for a server environment or service vm, that increase attack surface (zsh?)
 * things which really belong into seperate roles, such as editor / shell / git configuration
 * it installes locales-all instead of generating the required ones increasing  storage size immensily.
